### PR TITLE
Adding pet-linear output to deeplearning-prepare-data pipeline

### PR DESCRIPTION
**Pull request by ravih18**
Since pet-linear outputs are meant to be used for deep learning purpose, here is the PR to include pet-linear outputs to deeplearning-prepare-data pipeline.

`pet-linear` is a now a new option in the modality field. Options for pet images (acquisition label and suvr reference regions) were added to the CLI. The clinica reader function can now recognize pet-linear output files pattern to load them.

Documentation of deeplearning-prepare-data pipeline had also been updated.

Remaining tasks: update the tests for CI.

### CODE REVIEW

File: clinica/pipelines/deeplearning_prepare_data/deeplearning_prepare_data_cli.py [hunk 1]
```diff
@@ -35,9 +36,10 @@ def define_options(self):
             help="""For which modality the tensor will be extracted.
             't1-linear': images prepocessed with t1-linear pipeline.
             't1-extensive': images preprocessed with t1-extensive pipeline.
+            'pet-linear': images preprocessed with pet-linear pipeline.
             'custom': find images with a custom suffix in their filename and
             transform them to tensor format.""",
-            choices=["t1-linear", "t1-extensive", "custom"],
+            choices=["t1-linear", "t1-extensive", "custom", "pet-linear"],
```
Comment (alexandreroutier) [inline clinica/pipelines/deeplearning_prepare_data/deeplearning_prepare_data_cli.py:42]:
```suggestion
            choices=["t1-linear", "t1-extensive", "pet-linear", "custom"],
```

File: clinica/pipelines/deeplearning_prepare_data/deeplearning_prepare_data_cli.py [hunk 2]
```diff
@@ -105,6 +106,27 @@ def define_options(self):
             default="rgb",
         )
 
+        optional_pet = self._args.add_argument_group(
+            f"{Fore.BLUE}Pipeline options if you chose 'pet-linear' modality{Fore.RESET}"
```
Comment (alexandreroutier) [inline clinica/pipelines/deeplearning_prepare_data/deeplearning_prepare_data_cli.py:110]:
```suggestion
            f"{Fore.BLUE}Pipeline options if you chose 'pet-linear' pipeline{Fore.RESET}"
```

File: clinica/pipelines/deeplearning_prepare_data/deeplearning_prepare_data_cli.py [hunk 3]
```diff
@@ -105,6 +106,27 @@ def define_options(self):
...
+        )
+        optional_pet.add_argument(
+            "--acq_label",
+            help="""Name of the label given to the PET acquisition, specifying 
+            the tracer used (acq-<acq_label>). For instance it can be 'fdg' for 
+            fluorodésoxyglucose or 'av45' for florbetapir.""",
```
Comment (alexandreroutier) [inline clinica/pipelines/deeplearning_prepare_data/deeplearning_prepare_data_cli.py:116]:
```suggestion
            fluorodeoxyglucose or 'av45' for florbetapir.""",
```

File: clinica/pipelines/deeplearning_prepare_data/deeplearning_prepare_data_cli.py [hunk 4]
```diff
@@ -105,6 +106,27 @@ def define_options(self):
...
+            type=str,
+            default="",
+        )
+        optional_pet.add_argument(
+            "-suvr",
+            "--suvr_reference_region",
+            help="""Intensity normalization using the average PET uptake in reference regions 
+            resulting in a standardized uptake value ratio (SUVR) map. It can be 
+            cerebellumPons (used for amyloid tracers) or pons (used for 18F-FDG tracers).""",
```
Comment (alexandreroutier) [inline clinica/pipelines/deeplearning_prepare_data/deeplearning_prepare_data_cli.py:125]: No mentions of `cerebellumPons2` and `pons2` from your pipeline?

File: docs/Pipelines/DeepLearning_PrepareData.md [hunk 5]
```diff
@@ -6,10 +6,10 @@ deep learning library [[Paszke et al.,
...
 Three types of tensors are proposed: 3D images, 3D patches or 2D slices.
 
-Outputs from the [`t1-linear` pipeline](../T1_Linear) and [`t1-extensive`
-pipeline](https://clinicadl.readthedocs.io/en/latest/Preprocessing/T1_Extensive/)
-can be processed (for the moment, latter is available from
-[ClinicaDL](https://clinicadl.readthedocs.io).
+Outputs from the [`t1-linear` pipeline](../T1_Linear), [`t1-extensive`
+pipeline](https://clinicadl.readthedocs.io/en/latest/Preprocessing/T1_Extensive/) and 
+[`pet-linear` pipeline](../PET_Linear) can be processed (for the moment, 
+latter is available from [ClinicaDL](https://clinicadl.readthedocs.io).
```
Comment (alexandreroutier) [inline docs/Pipelines/DeepLearning_PrepareData.md:12]:
```suggestion
`t1-extensive` is available from [ClinicaDL](https://clinicadl.readthedocs.io).
```

File: clinica/pipelines/deeplearning_prepare_data/deeplearning_prepare_data_pipeline.py [hunk 6]
```diff
@@ -63,15 +64,21 @@ def build_input_node(self):
...
+                self.parameters.get("suvr_reference_region"),
+                self.parameters.get("use_uncropped_image"),
+            )
         if self.parameters.get("modality") == "custom":
             FILE_TYPE = {
                 "pattern": f"*{self.parameters.get('custom_suffix')}",
                 "description": "Custom suffix",
             }
 
-        # T1w_Linear file:
```
Comment (omar-rifai) [inline clinica/pipelines/deeplearning_prepare_data/deeplearning_prepare_data_pipeline.py:72]: Can you also change the comment on line 145/152 to reflect that change ?

**Pull request merged**